src/main/java/seedu/apollo/ui/Parser.java

```diff
@@ -136,10 +146,20 @@ private static Command parseCommand(String[] split, int size, ModuleList moduleD
             return new ExitCommand();
 
         case COMMAND_HELP_WORD:
-            if (!isOneWord(split)) {
+            if (!isOneWord(split) && (!split[1].equals("todo") &&!split[1].equals("deadline")&&!split[1].equals("event")
+                    &&!split[1].equals("mark")&&!split[1].equals("unmark")&&!split[1].equals("delete")
+                    &&!split[1].equals("find") &&!split[1].equals("date") &&!split[1].equals("addmod")
+                    &&!split[1].equals("delmod")&&!split[1].equals("week") &&!split[1].equals("showmod")
+                    &&!split[1].equals("listmod") &&!split[1].equals("list"))) {
+
+
                 throw new IllegalCommandException();
```

Comment:
Instead of this, consider using a switch case
ie. if (!isOneWord(split))
switch(split[1])
case "todo" : wtv
case "deadline" : wtv
...
default : throw new IllegalCommandException();